src/plugins/WebOfTrust/util/DelayedBackgroundJob.java

```diff
@@ -0,0 +1,26 @@
+package plugins.WebOfTrust.util;
+
+/**
+ * A {@link BackgroundJob} that is executed after triggers have been aggregated for some delay.
+ *
+ * @author bertm
+ */
+public interface DelayedBackgroundJob extends BackgroundJob {
+    /**
+     * Triggers the execution of the job after either aggregating triggers for at most some delay
+     * specified by the implementation or as soon as the currently running job finishes, whichever
+     * comes last. Implementations must ensure that this method is safe to invoke from any thread
+     * at any moment.
+     */
+    @Override
+    public void triggerExecution();
```

Comment:
As a final after-merger note on this:  Your explanation seems sound, the text seems correct from your point of view. I can perfectly live with the resulting javadoc. Sounds like this asks to be redefined somewhere in the future for slightly better comprehensibility for both regular users and mathematicians alike, but that is absolutely unimportant at this point.